mypy_django_plugin/transformers/manytomany.py

```diff
@@ -151,3 +151,57 @@ def get_model_from_expression(
         if model_info is not None:
             return Instance(model_info, [])
     return None
+
+
+def get_related_manager_and_model(ctx: MethodContext) -> Optional[tuple[Instance, Instance]]:
```

Comment:
You broke Python 3.8

```suggestion
def get_related_manager_and_model(ctx: MethodContext) -> Optional[Tuple[Instance, Instance]]:
```